dev/com.ibm.ws.wssecurity_fat.wsscxf/fat/src/com/ibm/ws/wssecurity/fat/cxf/sample/CxfSampleTests.java

```diff
@@ -142,8 +155,6 @@ public static void setUp() throws Exception {
         return;
     }
 
-    //4/2021 added allowed ffdc to run with EE8
-    @AllowedFFDC(value = { "java.net.MalformedURLException", "java.lang.ClassNotFoundException" })
```

Comment:
was `java.net.MalformedURLException` supposed to be removed here?

Reply:
If I remember correctly, it was tested by first commenting out the ffdc for ee9 testing, and it didn't prompt for ffdc failure.  therefore it's removed.  But will retest to confirm.

Reply:
It's confirmed this test doesn't generate ffdc with EE9 and only the other 2 tests (testEchoService, testEcho4Service) do.
This can be checked through messages.log.  If removing the allowedffdc from the 2 tests (testEchoService, testEcho4Service), these tests will fail.  
So no allowedffdc needed for this test.